recipeers/recetas.py

```diff
@@ -0,0 +1,11 @@
+from typing import List, Tuple
+from preparacion import Preparacion
+from ingrediente import Ingrediente
+
+class Receta:
+    def __init__(self, nombre: str, 
+                 listaIngredientes: List[Tuple[Ingrediente, float]], 
+                 pasosPreparacion: List[Preparacion]):
+        self.nombre = nombre
+        self.listaIngredientes = listaIngredientes
```

Comment:
O revisando las convenciones. Aquí estáis usando CamelCase sin embargo en el fichero de más arriba se usa snake_case.